src/Symfony/Component/Console/Formatter/OutputFormatterStyle.php

```diff
@@ -54,6 +54,7 @@ class OutputFormatterStyle implements OutputFormatterStyleInterface
     private $background;
     private $href;
     private $options = array();
+    private $handlesHrefGracefully;
```

Comment:
Nitpicking: would `$supportsHyperlinks` or `$supportsLinks` be a better variable name than `$handlesHrefGracefully`?

Reply:
Not really to me, as "supports" just means to me the terminal is able to render hyperlinks. 
Terminals not supporting it doesn't mean they will crash; so they handle it gracefully (ignoring the href), on the contrary of PhpStorm's one 😄